### DISCUSSION TITLE
unable to connect to server Indonesia

### DISCUSSION THREAD

**Discussion by 3xploiton3** [esnet/iperf#1753] [Q&A]
> iperf3 -c iperf.biznetnetworks.com -p 5201-5203

iperf3: error - unable to connect to server - server may have stopped running or use a different port, firewall issue, etc.: Connection refused

# Context

* Version of iperf3: iperf 3.17.1 (cJSON 1.7.15)

* Hardware:

Linux arch 6.10.5-zen1-1-zen #1 ZEN SMP PREEMPT_DYNAMIC Thu, 15 Aug 2024 00:25:05 +0000 x86_64
Optional features available: CPU affinity setting, IPv6 flow label, SCTP, TCP congestion algorithm setting, sendfile / zerocopy, socket pacing, authentication, bind to device, support IPv4 don't fragment, POSIX threads

any other alternative server for Indonesia location?

**Comment by davidBar-On** [comment]
As the error message say, probably the server is not running.  The public servers are not under the iperf3 team control, so you should probably try another public server. Alternatively retry tomorrow, as it may be that the server is under maintenance, which is usually done at the weekends.

By the way `-p` accepts only one port, so the right parameter should have been `-p 5201` or `-p 5202` or `-p 5203`.

**Reply by naim173** [reply]
How did you solve this Problem?
